packages/chart/src/Pie.ts

```diff
@@ -403,8 +406,12 @@ export interface Pie {
     startAngle(_: number): this;
     labelHeight(): number;
     labelHeight(_: number): this;
+    percentageFormat(): string;
+    percentageFormat(_: string): this;
+
 }
 Pie.prototype.publish("showSeriesPercentage", false, "boolean", "Append data series percentage next to label");
+Pie.prototype.publish("percentageFormat", ",.0f", "string", "Number format used for formatting values", null, {disable: w => !w.showSeriesPercentage()});
```

Comment:
Rename to "seriesPercentageFormat" and change description to mention "series" as well.